curriculum/challenges/english/07-scientific-computing-with-python/build-an-arithmetic-formatter-project/arithmetic-formatter.md

```diff
@@ -1,9 +1,9 @@
 ---
 id: 5e44412c903586ffb414c94c
-title: Arithmetic Formatter
+title: Build an Arithmetic Formatter Project
 challengeType: 23
```

Comment:
I wonder if it needs a different value of `challengeType` to avoid the instructions from disappearing